### ISSUE TITLE
Cant Access the dashboard

### ISSUE THREAD

**Issue by gautamkrishnar** [joshsoftware/code-curiosity#480]
See the screenshot below:
<img width="1440" alt="Screenshot 2019-05-19 at 12 11 24 AM" src="https://user-images.githubusercontent.com/8397274/57973700-a9c31580-79ca-11e9-8f50-4477c071f8ad.png">

**Comment by wyhaines** [comment]
Closing. Not relevant now.

**Comment by gautamkrishnar** [comment]
@wyhaines can you please let me know what happened to the project? Did JoshSoftware stopped supporting this?

**Comment by wyhaines** [comment]
Hi @gautamkrishnar .  I should have been a little more forward thinking when we started the server up again, that people might find it before it was ready to be used again. So, apologies if the issue comments etc. seemed a little abrupt.

I can't speak to the details of what caused Code Curiosity to be shut down, as I wasn't with Josh or involved with Code Curiosity at the time. However, I and Josh Software are in the process of fixing things and bringing it back to life.

The very tentative timeline is to bring it back into the public eye around the end of the year. We are trimming the backlog of issues as it looks like I inherited about three years of them, so that we can focus on the highest priority items.

I will check the logs for your error and see if it's something that I do need to address now. If it is, I'll open a new issue on it and come back to you if I need more information.

Thank you. Don't hesitate to speak up if you have any more questions.
